webdevbroz/pomodoro-app
# Add design system

## Goal
We want to add the colors, fonts, and assets provided to us for building this project. As well as the UI components we will be using. https://ui.shadcn.com/ in our case.

The idea is to add our custom global styles through tailwind so that we can use them as utility classes throughout the application.

Related issue: https://github.com/webdevbroz/pomodoro-app/issues/5

## How to test
- Check that the global font `kumbh-sans` is applied to the `Place Holder` text.
- Check that the background color `#1e213f` is applied.
- Check to see that the Button component added for testing works.

## Screenshots
